apps/meteor/ee/server/apps/communication/rest.ts

```diff
@@ -1120,24 +1122,46 @@ export class AppsRestApi {
 					return API.v1.notFound(`No App found by the id of: ${this.urlParams.id}`);
 				},
 				async post() {
-					if (!this.bodyParams.status || typeof this.bodyParams.status !== 'string') {
+					const { id: appId } = this.urlParams;
+					const { status } = this.bodyParams;
+
+					if (!status || typeof status !== 'string') {
 						return API.v1.failure('Invalid status provided, it must be "status" field and a string.');
 					}
 
-					const prl = manager.getOneById(this.urlParams.id);
-
+					const prl = manager.getOneById(appId);
```

Comment:
Can we use a better naming for this variable? prl is not giving the idea of what this is responsible for.